### PR TITLE
feat(docs): add Browserstack information to README

### PR DESCRIPTION
**Pull request by lievenvg**
## PR Checklist

This PR fulfills the following requirements:

- [x] The commit message follows our guidelines: [Contributing guidelines](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] A [changelog entry](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/guidelines/CHANGELOG.md) was added

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Issue Number: N/A

## What is the new behavior?
To apply for the BrowserStack free open source program, BrowserStack asks the following in return:
 - A shoutout to BrowserStack on our GitHub project page or website
 - Post and hyperlink the BrowserStack logo
 - Add a line about how we use BrowserStack

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,10 +76,19 @@ This library contains the following components:
...
+### Test automation
 
 If you want to start automating your tests, you can have a look at our [Test Automation Guide](./TEST_AUTOMATION.md) for some tips on how to use data-attributes to keep your tests stable and maintainable.
 
+### Cross Browser Testing
+
+We are using [Browserstack Live](https://www.browserstack.com/live) to make sure that our components work correctly on all major browsers and platforms.<br/>
+The people at Browserstack kindly offer an unlimited testing program, free of charge for open source projects so a big thanks to them!
+
+<a href="http://browserstack.com/"><img width="250" src="./assets/browserstack-logo-600x205.png" alt="Browserstack logo"></a>
```
Comment (TriangleJuice) [inline README.md]:
```suggestion
<a href="http://browserstack.com/"><img width="250" src="assets/browserstack-logo-600x205.png" alt="Browserstack logo"></a>
```

Comment (TriangleJuice) [reply README.md]: Or can't we take the logo from their CDN?

Comment (lievenvg) [reply README.md]: Replaced the url with a GitHub CDN.

Comment (lievenvg) [reply README.md]: @TriangleJuice 
Is this update ok for you?

Comment (TriangleJuice) [reply README.md]: Yes, we will add this in the next update of ACPaaS UI.

**Pull request merged**